src/Controls/src/Build.Tasks/CompiledValueProviders/SetterValueProvider.cs

```diff
@@ -23,8 +23,9 @@ public IEnumerable<Instruction> ProvideValue(VariableDefinitionReference vardefr
 			if (SetterValueIsCollection(bpRef, module, node, context))
 				yield break;
 
+			// valueNode is null, for example, when OnPlatform doesn't have a match for the current platform, so the property should not be set
 			if (valueNode == null)
-				throw new XamlParseException("Missing Value for Setter", (IXmlLineInfo)node);
+				yield break;
```

Comment:
I'm afraid this will have effects outside of OnPlatform

Reply:
@StephaneDelcroix  - Yes, this change can have effects outside of OnPlatform, but previously those cases failed with the exception here, so the risk of breaking anything, making things worse, seems small - it previously just error'ed out. That said, if you can tell me how this code is actually used outside of OnPlatform, like suggest some test scenarios, I can test those.

Reply:
The property would not be set, right?. So for example, here:

```
<Style x:Key="ButtonStyle0" TargetType="Button">
    <Setter Property="FontSize" Value="{OnPlatform iOS=36}" />
</Style>
```

The FontSize property on Android, would  have the default value?

Reply:
Right